lib/__tests__/test.js

```diff
@@ -14,7 +14,7 @@ test(`Should throw an error if config has error`, () => {
 	const pluginRun = postcss([plugin(opts)])
 		.process('', { from: undefined })
 		.then(() => {
-			expect('Plugin should throw an error').toBeFalst();
+			expect('Plugin should throw an error').toBeFalsy();
```

Comment:
Fixing typo
This test passes even if you put .toBeTruthy() there so there's something wrong with it 😵

Reply:
This line should never be executed, because plugin throws an error, and it goes directly to `catch`. Idea was that if this line shows an error, then plugin didn't do what it suppose to do.

Probably could remove whole `.then()`, since line 23 should just work without it.